### PR TITLE
Remove old and irrelevant files

### PR DESCRIPTION
**Pull request by Gnappuraz**
This PR removes what I think is either outdated or not relevant for the repo anymore. Especially for `spec_v1.0.md` it contains outdated informations that can be quite misleading, I will create a new and up to date version of it. As for `/particl/create_tx.md` I think is not relevant at all at this point and should simply be removed.

### CODE REVIEW

**Comment by castarco** [comment]
ACK 983f41092a57d2bd03400714a7787b6a06cd9f8a

File: specs/spec_v1.0.md [hunk 1]
```diff
@@ -1,322 +0,0 @@
-# **Summary**
```
Comment (kostyantyn) [inline specs/spec_v1.0.md]: I don't mind to remove this file but as I recall @cmihai used it for the reference. Would be good to ask his opinion.

**Comment by cmihai** [comment]
Concept ACK, but signal_callback_system.md still seems relevant?

File: specs/spec_v1.0.md (hunk 1 above)
Comment (cmihai) [reply specs/spec_v1.0.md]: That was just to understand the commit messages. I don't need the spec right now.

**Comment by AM5800** [comment]
I join @cmihai what is wrong with `signal_callback_system.md`?

**Comment by Gnappuraz** [comment]
That `signal_callback_system.md` is incomplete and incorrect in some parts. Also is more bitcoin internals than stuff relevant to us. I'd rather remove it cause on its own also is a bit silly honestly, and not so useful.

**Comment by cornelius** [comment]
The esperanza spec is linked from the [Unit-e README](https://github.com/dtr-org/unit-e/blob/master/README.md#the-unit-e-client). This needs to be corrected then as well. Would be good to have this information in the docs. We have an open issue for that (https://github.com/dtr-org/docs.unit-e.io/issues/22).

**Comment by Gnappuraz** [comment]
@cornelius I'd rather get rid of it now, update it asap and republish it. Atm is really outdated on some parts, people reading it would just be confused I guess.

**Comment by cornelius** [comment]
@Gnappuraz I'm fine with removing the files now as long as we make sure we don't end up with broken links in the unit-e README.

**Comment by Gnappuraz** [comment]
@cornelius I created a PR on unit-e to get rid of the links.

**Pull request merged**